cli/command/container/cp.go

```diff
@@ -71,6 +105,7 @@ func NewCopyCommand(dockerCli command.Cli) *cobra.Command {
 	flags := cmd.Flags()
 	flags.BoolVarP(&opts.followLink, "follow-link", "L", false, "Always follow symbol link in SRC_PATH")
 	flags.BoolVarP(&opts.copyUIDGID, "archive", "a", false, "Archive mode (copy all uid/gid information)")
+	flags.BoolVarP(&opts.quiet, "quiet", "q", false, "Does not display copying process")
```

Comment:
What do you think if we set the default based on if stdout is a tty?

Reply:
Could be a good idea.

Reply:
Yep.

Reply:
Ok!

Reply:
Done @cpuguy83